op-service/sources/testdata/data/blocks/post-shanghai-bad-transactions-hash_metadata.json

```diff
@@ -0,0 +1 @@
+{"name":"post-shanghai-bad-transactions-hash","fail":true, "reason":"failed to verify block hash"}
```

Comment:
We're still reading in and executing these block tests though - just they've been manually tweaked to make sure the block hash is valid so they get through to the actual part of code they were intending to test.

Reply:
We're still using the contents of the blocks directory - it's just been tweaked to actually test what we expected it to instead of always failing on the block hash not matching.  The generator script no longer generates the block data because its not smart enough to get them right and we don't want it to overwrite them.